playwright-e2e/utils/date-utils.ts

```diff
@@ -15,14 +15,23 @@ export function dateFormat(timeZone?: string): Intl.DateTimeFormat {
     });
 }
 
+export function getToday(): Date {
+  return normalizeDate(new Date());
+}
+
+const normalizeDate = function(date: Date) {
+  date.setMinutes(date.getMinutes() + date.getTimezoneOffset());
+  return date;
+};
```

Comment:
With this new function, I'm hoping to resolve an long standing issue with dates. Some tests fail due to date of month that is one-day off.